packages/atomic/scripts/process-css.mjs

```diff
@@ -128,3 +136,80 @@ export async function processCssFiles(srcDir, distDir) {
     }
   }
 }
+
+function getAllJsFiles(dir) {
+  let results = [];
+  for (const name of readdirSync(dir)) {
+    const p = join(dir, name);
```

Comment:
I guess `p` stands for `path`. How about renaming the variable to `path` instead? 😁